api/v1alpha1/ionoscloudmachine_types.go

```diff
@@ -168,6 +169,18 @@ type IonosCloudMachineSpec struct {
 	//+optional
 	FailoverIP *string `json:"failoverIP,omitempty"`
 
+	// NetworkID is the ID of the LAN if an existing LAN is required.
+	// A typical use-case is when an existing LAN has a Failover Group and the FailoverIP configured.
+	// This field is used when the Failover Group is created in another LAN.
+	//
+	// If the machine is a control plane machine, this field will not be taken into account.
```

Comment:
Sorry for not noticing earlier, but I think that last sentence is wrong. I don't see any logic discarding this field when handling a control plane machine.

Reply:
you are actually right, let me check why I added this comment.